sample/ERC000038.json

```diff
@@ -0,0 +1,589 @@
+        "chemical compound": {
+          "type": "array",
+          "items": {
+            "required": [
+              "value"
+            ],
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            }
+          },
+          "description": "A drug, solvent, chemical, etc., with a property that can be measured such as concentration (http://purl.obolibrary.org/obo/CHEBI_37577).",
+          "minItems": 1
+        },
+        "Depth": {
+          "minItems": 1,
+          "type": "array",
+          "maxItems": 1,
+          "items": {
+            "required": [
+              "value",
+              "units"
+            ],
+            "properties": {
+              "units": {
+                "enum": [
+                  "m"
+                ]
+              },
+              "value": {
+                "pattern": "([0-9]*\\.)?[0-9]+\\s?([-]\\s?([0-9]*\\.)?[0-9]+)?",
+                "type": "string"
+              }
+            }
+          },
+          "description": "The distance below the surface of the water at which a measurement was made or a sample was collected. Format: ####.##, Positive below the sea surface. SDN:P06:46:ULAA for m. Example: 14.71"
+        },
+        "environment (feature)": {
+          "minItems": 1,
+          "type": "array",
+          "description": "Environmental feature level includes geographic environmental features. Compared to biome, feature is a descriptor of the more local environment. Examples include: harbor, cliff, or lake. EnvO (v 2013-06-14) terms can be found via the link: www.environmentontology.org/Browse-EnvO",
+          "items": {
+            "required": [
+              "value"
+            ],
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "Sampling Platform": {
+          "items": {
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            },
+            "required": [
+              "value"
+            ]
+          },
+          "description": "Refers to the unique stage from which the sampling device has been deployed. Includes Platform category from SDN:L06, http://seadatanet.maris2.nl/v_bodc_vocab_v2/search.asp?lib=L06, and Platform name. Example: Research Vessel Tara.",
+          "type": "array",
+          "maxItems": 1,
+          "minItems": 1
+        },
+        "storage conditions (fresh/frozen/other)": {
+          "description": "explain how and for how long the soil sample was stored before DNA extraction.",
+          "items": {
+            "required": [
+              "value"
+            ],
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            }
+          },
+          "maxItems": 1,
+          "type": "array",
+          "minItems": 1
+        }
+      }
+    }
+  }
+}
```

Comment:
The `sampleRelationships` definition is also missing, see [here](https://github.com/EMBL-EBI-SUBS/validation-schemas/blob/master/sample/sample-schema.json#L65).